openrtx/src/protocols/M17/M17LinkSetupFrame.cpp

```diff
@@ -76,6 +76,12 @@ void M17LinkSetupFrame::setType(streamType_t type)
     data.type  = type;
 }
 
+// Added by KD0OSS (2025)
```

Comment:
This is adding meta text, but this PR deals with GNSS data. I'd remove these changes and add them to the other incoming PRs

Reply:
Thanks, I can move this out!